week-2-practise/level-2/4-slice.js

```diff
@@ -4,8 +4,9 @@
 // - return a new array with the item removed
 // - remove the item at the specified index
 
-function remove(arr, index) {
-  return; // complete this statement
+function remove(arr, index) { 
+  arr.splice(index,index);
```

Comment:
Not quite correct here. For example, if you change numbers to [1,2,3,4], this does something unexpected. Can you see why? Read the documentation for the splice method carefully